tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDKUtil.java

```diff
@@ -309,6 +315,31 @@ public static void openAsProject( SDK sdk )
                 }
 
                 sdkProject.open( npm );
+
+                IFilterMatcherDescriptor descriptor = null;
+                IFilterMatcherDescriptor[] descriptors = ResourcesPlugin.getWorkspace().getFilterMatcherDescriptors();
+
+                for( int i = 0; i < descriptors.length; i++ )
+                {
+                    if( descriptors[i].getName().equals( "AND" ) )
+                    {
+                        descriptor = descriptors[i];
+                    }
+                }
+
+                if( descriptor != null )
+                {
+                    FileInfoMatcherDescription fmd = new FileInfoMatcherDescription( descriptor.getId(), null );
+
+                    IFolder[] folders = { sdkProject.getFolder( "portlets" ), sdkProject.getFolder( "layouttpl" ),
```

Comment:
This is not the right way to do this.  This iwll be confusing to a user that imports the SDK project and doesn't see the projects listed in the folder.  

However, as each project is imported individually we should add a filter for that project specifically.  

So at first there is no filters when you import SDK projects.  But when each individual project is imported later, we need to add one single filter for each project, one at a time.